### ISSUE TITLE
Menu Required Permission don't work

### ISSUE THREAD

**Issue by OGreenC** [BG-Software-LLC/SuperiorSkyblock2#1140]
### Minecraft's Version

paper-1.17.1-411.jar

### Plugin's Version

SuperiorSkyblock2-1.10.0-b646.jar and SuperiorSkyblock2-1.10.0.jar (both tested)

### Describe the bug

I'm experiencing the same error, as seen in issue #1138 - I've attempted using both the permission and required permission setting.

In fact, my setup worked in previous versions of the plugin.

Permission requirements was removed from all default menus in the following commit:  https://github.com/BG-Software-LLC/SuperiorSkyblock2/commit/bb8369e7e82f1d26d80eefce3f0e09362fcd181c

But I can't seem to find anywhere, where it is described, that it is no longer possible - Only that it has been removed from the default menus.

Please see below, for an example of an item:
```items:
#Medlem
  'a':
    schematic: 'MedlemNormal'
    required-permission: 'island.medlem'
    biome: PLAINS
    access:
      type: PLAYER_HEAD
      skull: 'eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvYzk1ZDM3OTkzZTU5NDA4MjY3ODQ3MmJmOWQ4NjgyMzQxM2MyNTBkNDMzMmEyYzdkOGM1MmRlNDk3NmIzNjIifX19'
      name: '&2&lMEDLEM &aNormal Island &a(&l✔&a)'
      lore:
        - '&7Skyblock øen som vi kender og'
        - '&7elsker den!'
        - '&7 '
        - '&7Biome: &aPlains'
        - '&7&o(&eLegends &7og &bTitans &7kan'
        - '&7&oskifte biomes efterfølgende&7&0)'
        - '&7 '
        - '&7&o(( &f&oHøjreklik &7&ofor at se øen. ))'
    no-access:
      type: PLAYER_HEAD
      skull: 'eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZjliYTdmZWY2YTFhOGJkODk5YWJhZTRhNWI1NGNiMGVjZTUzYmFkYzY3N2MxNjY4YmVlMGE0NjIxYTgifX19'
      name: '&2&lMEDLEM &aNormal Island &c(&l✖&c)'
      lore:
        - '&7Skyblock øen som vi kender og'
        - '&7elsker den!'
        - '&7 '
        - '&7Biome: &aPlains'
        - '&7&o(&eLegends &7og &bTitans &7kan'
        - '&7&oskifte biomes efterfølgende&7&0)'
        - '&7 '
        - '&7&o(( &f&oHøjreklik &7&ofor at se øen. ))'
 ```
        
To be clear - This configuation has worked in the past, but stopped working, after I updated the plugin to the most recent release.

I've also made sure to test it on a smurf, with no permissions, besides the nessecary permissions needed, to open the island creation menu.

I've also tried `permission` instead of `required-permission` as shown above, as mentioned in the wiki - The problem still persists.
### To Reproduce

Create a menu, with an item using the `required-permission` tag, in the newest release of SuperiorSkyblock.
(Or simply use the old default file for island creation, before removal of such)

It will be apparent, that permission-requirements wont take effect.

**Comment by OmerBenGera** [comment]
Hey, please follow this tutorial:
<https://wiki.bg-software.com/superiorskyblock/overview/menus#permissions-section>

I added an example on how to use it.
If it still doesn't work, upload your menu here

**Comment by OmerBenGera** [comment]
Does it work for you?

**Comment by OGreenC** [comment]
Oh shoot!
Sorry for the late response - I usually get a notification when i get a respond on an Issue, but somehow this didn't happen this time.

I've now look on, and implemented the provided example, and I realized, that the permissions had been moved from being a part of the item itself, to having it's own section in the menu, as it is done with sounds.

With this discovery, I managed to make it work - Thank you for the help!
